types/txMsg.ts

```diff
@@ -1,11 +1,18 @@
-export type TxType = "send" | "delegate" | "undelegate" | "redelegate" | "claimRewards";
+export type TxType =
+  | "send"
+  | "delegate"
+  | "undelegate"
+  | "redelegate"
+  | "claimRewards"
+  | "setWithdrawAddress";
```

Comment:
With this we can create a `function gasOfMsg(msg: TxType): number`. Then we can say by default each of them get 100_000 gas (can be changed later) and we calculate the gas by saying
- 100_000 gas for the signature verification (flat, once per transaction)
- Loop over the messages and add `gasOfMsg(msgType)` for each message.

This would make the gas management more maintainable and clear.